.github/workflows/build.yml

```diff
@@ -23,6 +23,7 @@ jobs:
       STAGING_CLOUDFRONT_DISTRIBUTION_ID: E37K3V5Y65XQIX
       REVIEW_CLOUDFRONT_DISTRIBUTION_ID: E3KUGPF02I4CJ4
       VITE_FOUNDATION_BUILD: ${{ github.repository_owner == 'microbit-foundation' }}
+      VITE_SENTRY_DSN: ${{ secrets.VITE_SENTRY_DSN }}
```

Comment:
```suggestion
      # Not really secret but worth keeping out the repo to prevent unintended use
      VITE_SENTRY_DSN: ${{ secrets.SENTRY_DSN }}
```